scene_synchronizer.h

```diff
@@ -540,7 +540,7 @@ class ClientSynchronizer : public Synchronizer {
 
 	RBSet<EndSyncEvent> sync_end_events;
 
-	struct DeferredSyncInterpolationData {
+	struct trickledSyncInterpolationData {
```

Comment:
This should be `TrickledSyncInterpolationData` instead.

Reply:
Now done in https://github.com/GameNetworking/network_synchronizer/compare/78c8bdb008f93ec9447d130061ed5d8a4678184a..074a9f75b5f01b743d7bda324935eb769d1c374d